exopy/utils/widgets/qt_tree_widget.py

```diff
@@ -783,8 +783,7 @@ def _on_nid_changed(self, nid, col):
         if new_label != old_label:
             if new_label != '':
                 node.exit_rename(obj, new_label)
-            else:
-                self._set_label(nid, old_label)
+            self._set_label(nid, node.get_label(obj))
```

Comment:
Why this change ?

Reply:
When the new name is invalid (because it's not unique), exit_rename doesn't set obj.name to new_label. The issue is, without this patch, the name displayed in the tree would be stuck on the new_label, even though it's invalid and doesn't reflect the actual name.
The systematic call to _set_label syncs the two.